docs/05-audio-interfaces.md

```diff
@@ -16,60 +16,77 @@ USB soundcards are also supported without any special configuration needed. Just
 
 ## DAC boards
 
+### Overview
+
+balenaSound supports a wide variety of DACs, but on an ever-growing market chances are your particular DAC might not work with the project just yet. Broadly speaking it's a two step process in order for a DAC to be officially supported: 
+1) the required kernel driver must be available in balenaOS 
+2) adequate testing must be performed with balenaSound
+
+Latest versions of balenaOS already ship with drivers for most if not all of the DACs available. The testing story however is different, it's not possible for us to test all of them individually as it would take an incredible amount of resources. For this reason we rely on community contributions to help catalogue existing DACs into three categories:
```

Comment:
```suggestion
Latest versions of balenaOS already ship with drivers for most, if not all, of the DACs available. The testing story, however, is different. It's not possible for us to test all of them individually as it would take an incredible amount of resources. For this reason, we rely on community contributions to help catalogue existing DACs into three categories:
```